app/(dashboard)/mailboxes/[mailbox_slug]/[category]/conversation/conversationHeader.tsx

```
@@ -0,0 +1,118 @@
+import {
+    ChevronLeft,
+    ChevronRight,
+    Info,
+    Link as LinkIcon,
+    PanelRightClose,
+    PanelRightOpen,
+    X,
+} from "lucide-react";
```

ESLint

[error] Replace `⏎····ChevronLeft,⏎····ChevronRight,⏎····Info,⏎····Link·as·LinkIcon,⏎····PanelRightClose,⏎····PanelRightOpen,⏎····X,⏎` with `·ChevronLeft,·ChevronRight,·Info,·Link·as·LinkIcon,·PanelRightClose,·PanelRightOpen,·X·`

(prettier/prettier)